eventbrite.py

```diff
@@ -0,0 +1,380 @@
+      "hidden": false,
+      "auto_hide": false,
+      "auto_hide_before": "",
+      "auto_hide_after": "",
+      "sales_channels": ["online","atd"],
+      "hide_sale_dates": false,
+      "delivery_methods": ["electronic"],
+      "include_fee": false,
+      "sales_start": "2023-11-09T08:00:00Z",
+      "sales_end": "2023-11-11T03:00:00Z"
+    }
+}
+"""
+
+example_free_ticket_class = """
+{
+    "ticket_class": {
+      "name": "API Test0",
+      "free": true,
+      "donation": false,
+      "quantity_total": "100",
+      "minimum_quantity": "1",
+      "maximum_quantity": "10",
+      "hidden": false,
+      "auto_hide": false,
+      "auto_hide_before": "",
+      "auto_hide_after": "",
+      "sales_channels": ["online","atd"],
+      "hide_sale_dates": false,
+      "delivery_methods": ["electronic"],
+      "include_fee": false,
+      "sales_start": "2023-11-09T08:00:00Z",
+      "sales_end": "2023-11-11T03:00:00Z"
+    }
+}
+"""
+
+def get_organization(http):
+    url = f"{EVENTBRITE_API_BASE_URL}users/me/organizations/"
+    headers = {
+        'Authorization': eventbrite_api_token
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status == 200:
+        data = resp.data.decode('utf-8')
+        print(data)
+    else:
+        print(f"HTTP Error: {resp.status}")
+
+def get_event_by_id(http, event_id):
+    url = f"{EVENTBRITE_API_BASE_URL}events/{event_id}/"
+    headers = {
+        'Authorization': eventbrite_api_token
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status == 200:
+        data = resp.data.decode('utf-8')
+        print(data)
+    else:
+        print(f"HTTP Error: {resp.status}")
+
+def get_events_by_organization(http, organization_id):
+    url = f"{EVENTBRITE_API_BASE_URL}organizations/{organization_id}/events"
+    headers = {
+        'Authorization': eventbrite_api_token
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status == 200:
+        data = resp.data.decode('utf-8')
+        print(data)
+    else:
+        print(f"HTTP Error: {resp.status}")
+
+def create_event(http, organization_id ,event_details):
+    url = f"{EVENTBRITE_API_BASE_URL}organizations/{organization_id}/events/"
+    headers = {
+    'Authorization': eventbrite_api_token,
+    'Content-Type': 'application/json'
+    }
+
+    try:
+        # Make the POST request
+        resp = http.request('POST', url, headers=headers, body=event_details)
+        # Check the status code to ensure a successful request
+        if resp.status == 200:
+            data = resp.data.decode('utf-8')
+            print(data)
+            return json.loads(data)
+        elif resp.status == 400:
```

Comment:
Remove conditions checking of 200 and 400, assume it's going to be 200. If it's anything but 200 then I want exception to occur.